src/components/interaction-designer/toolbar/UndoRedoButtonGroup.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="undo-redo-button-group btn-group">
+    <button
+      class="btn btn-sm btn-outline-primary"
+      :disabled="!canUndo"
+      @click.stop="handleUndo">
+      <font-awesome-icon :icon="['fas', 'undo']" />
+      <!--v-b-tooltip.hover="undoTooltip"-->
+    </button>
+    <button
+      class="btn btn-sm btn-outline-primary"
+      :disabled="!canRedo"
+      @click.stop="handleRedo">
+      <font-awesome-icon :icon="['fas', 'redo']" />
+      <!--v-b-tooltip.hover="redoTooltip"-->
+    </button>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {computed} from 'vue'
+import {useStore} from '@/store/useStore'
+
+const store = useStore()
+
+const canUndo = computed<boolean>(() => store.getters['undoRedo/canUndo'])
+const canRedo = computed<boolean>(() => store.getters['undoRedo/canRedo'])
+
+// const undoTooltip = computed<string | undefined>(() => store.getters['undoRedo/undoTooltip'])
```

Comment:
I think designer has agreed on texts here? We can just use localized texts

- [ ] enable the tooltip

Reply:
Sorry! I forgot to push the latest change. Updated now. Just 2 little commits:
https://github.com/FLOIP/flow-builder/pull/459/commits/5b118e9242a0d7caa1a8e027ebbf72e7045d1f2a
https://github.com/FLOIP/flow-builder/pull/459/commits/2da74b8367043237c16259fafa27ed5ab65eb9de